packages/components/.eslintrc.cjs

```diff
@@ -0,0 +1,13 @@
+/** @type {import("eslint").Linter.Config} */
+module.exports = {
+  root: true,
+  extends: [require.resolve('@srcbook/configs/eslint/library.js')],
```

Comment:
should use `@srcbook/configs/eslint/react.js` otherwise the accessibility (a11y) rules and the react-hook rules will be missed and jest types may break